github/github_test.go

```diff
@@ -105,6 +105,16 @@ func testHeader(t *testing.T, r *http.Request, header string, want string) {
 	}
 }
 
+func testCanonicalHeader(t *testing.T, r *http.Request, header string, want string) {
+	canonical := http.CanonicalHeaderKey(header)
+	for _, h := range r.Header[canonical] {
+		if h == want {
+			return
+		}
+	}
+	t.Errorf("Multiple header get, wanted %s but not found for header %s", want, header)
+}
```

Comment:
Good question.

The reason I added this was because I found when testing, that I needed to check for multiple accept headers. So I found out the preferred way to test that is through the canonical form on the Go docs for [Header](https://golang.org/pkg/net/http/#Header).